cpp/src/Ice/DefaultsAndOverrides.cpp

```diff
@@ -65,8 +65,24 @@ IceInternal::DefaultsAndOverrides::DefaultsAndOverrides(const PropertiesPtr& pro
 
     const_cast<chrono::milliseconds&>(defaultInvocationTimeout) =
         chrono::milliseconds(properties->getIcePropertyAsInt("Ice.Default.InvocationTimeout"));
+    if (defaultInvocationTimeout.count() < 1 && defaultInvocationTimeout.count() != -1)
+    {
+        throw InitializationException(
+            __FILE__,
+            __LINE__,
+            "invalid value for Ice.Default.InvocationTimeout " + to_string(defaultInvocationTimeout.count()) + "'");
```

Comment:
I would add a colon and remove the quotes, as in:
```
invalid value for Ice.Default.InvocationTimeout: -5
```

Same in the other languages, including JS.

Reply:
Fixed